promlinter.go

```diff
@@ -395,17 +483,50 @@ func (v *visitor) parseOptsExpr(n ast.Node) (*opt, *string) {
 		return v.parseOptsExpr(stmt.X)
 	}
 
-	return nil, nil
+	return nil
 }
 
-func (v *visitor) parseCompositeOpts(stmt *ast.CompositeLit) (*opt, *string) {
+func (v *visitor) parseCompositeOpts(stmt *ast.CompositeLit) *opt {
 	metricOption := &opt{}
-	var help *string
+
 	for _, elt := range stmt.Elts {
+
+		// labels
+		label, ok := elt.(*ast.BasicLit)
+		if ok {
+			metricOption.labels = append(metricOption.labels, strings.Trim(label.Value, `"`))
+			continue
+			// go down
```

Comment:
Do we need this comment?

Reply:
No, this may a noise comment, just removed.